roles/openshift_node_upgrade/tasks/registry_auth.yml

```diff
@@ -0,0 +1,24 @@
+---
```

Comment:
Same comment as in #5522 - any way to be more DRY and not require duplicates of this file hanging around?

Reply:
Replied to #5522 Plan on de-duplicating these items next cycle.